content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+        </tr>
+    </tbody>
+</table>
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy an Istio control plane with the following steps:
+
+1. To pass configuration values to the Istio operator for installation,
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+{{< text bash >}}
+$ cat <<EOF> ${WORK_DIR}/${CLUSTER_2}/install.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: ${MESH}
+      multiCluster:
+        clusterName: ${CLUSTER_2}
+      network: ${NETWORK_1}
+
+      meshNetworks:
+        ${NETWORK_1}:
+          endpoints:
+          # Always use ‘Kubernetes' as the registry name for
+          # endpoints that reside on the same network as this
+          # instance of the Istio control plane.
+          - fromRegistry: Kubernetes
+          gateways:
+          - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+      # Expose the control plane through istio-ingressgateway.
+      meshExpansion:
+        enabled: true
+EOF
+{{< /text >}}
+
+1. Install Istio on Cluster-2 with the following command:
+
+    {{< text bash >}}
+    $ istioctl --context=${CTX_2} manifest apply -f \
+    ${WORK_DIR}/${CLUSTER_2}/install.yaml
+    {{< /text >}}
+
+1. Verify that the control plane is ready with the following command:
+
+    {{< text bash >}}
+    $ kubectl --context=${CTX_2} -n istio-system get pod
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    istiod-f756bbfc4-thkmk                  1/1     Running   0          136m
+    prometheus-b54c6f66b-q8hbt              2/2     Running   0          136m
+    {{< /text >}}
+
+1. After the status of all pods is `Running`, you can continue configuring
+   your deployment.
+
+**Congratulations!**
+
+You successfully added a control plane to your primary
+cluster.
+
+Next, configure endpoint discovery to support cross-cluster load balancing.
+
+## Configure endpoint discovery
+
+To enable cross-cluster load balancing in your mesh, configure endpoint
+discovery. This feature requires that clusters share secrets between them. If
+the shared secrets provide the [needed trust](#heading=h.ww2golqkax84), each
+cluster in the mesh can access the API server in the other clusters directly.
+Using the previously set environment variables, configure endpoint discovery
+with the following steps:
+
+1. Share the secret of `Cluster_1` with `Cluster_2`:
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_1} \
+  --name=${CLUSTER_1} | \
+  kubectl apply -f - --context=${CTX_2}
+{{< /text >}}
+
+1. Share the secret of `Cluster_2` with `Cluster_1`:
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_2} \
+  --name=${CLUSTER_2} | \
+  kubectl apply -f - --context=${CTX_1}
+{{< /text >}}
```

Comment:
What do I do when I have more than 2 primary clusters?  Do I share the secrets with all of them?  Does this explode quickly?

Reply:
Yes, we have to do that today. We should say that here.

Reply:
Added a tip with that information.